### PR TITLE
docs: update migration message

### PR DESCRIPTION
**Pull request by wenytang-ms**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/11209305

1. since ARM and Multi-Env feature enabled, toolkit need to migration project, otherwise, the project can not develop and use in the future.
2. during migration, .fx folder structure will changed, and create new ARM templates, Other unchanged.
3. it's better to backup the .fx folder, since migration is irreversible. For the backup method, consider using git.
4. once user click cancel, toolkit will continue to ask for migration when trigger teamsfx command. Unless  downgrade the toolkit or disable feature flags.

Message changes as

**We have detected that your toolkit supports ARM and Multi-Env features. In order to support the development and use of the project in the future, We will migrate your project. Migration will change the folder structure of the .fx, please backup this folder before migration, and we will create ARM templates for new version.**

### CODE REVIEW

**Comment by timngmsft** [comment]
I don't think we have to tell people why their project will be migrated. This is a newer version of the toolkit; so we should migrate people along. @pierceboggan thoughts? This would be my proposal:

Teams Toolkit will migrate your project to support the latest features in Teams Toolkit 2.7.2. This process will change the contents of the '.fx' folder. Backup this folder or put your project in source control if you want to revert your project back to Teams Toolkit 2.7.1.

(can you check that the version tags are correct?)

**Comment by wenytang-ms** [comment]
> I don't think we have to tell people why their project will be migrated. This is a newer version of the toolkit; so we should migrate people along. @pierceboggan thoughts? This would be my proposal:
> 
> Teams Toolkit will migrate your project to support the latest features in Teams Toolkit 2.7.2. This process will change the contents of the '.fx' folder. Backup this folder or put your project in source control if you want to revert your project back to Teams Toolkit 2.7.1.
> 
> (can you check that the version tags are correct?)

Thank you @timngmsft for your feedback. the message has updated, and the version start 2.7.0 will migration the project

**We will migrate your project to support the latest features in Teams Toolkit. This process will change the contents of the '.fx' folder. Backup this folder or put your project in source control if you want to revert your project back to Teams Toolkit version below 2.7.0**

**Comment by adashen** [comment]
> > I don't think we have to tell people why their project will be migrated. This is a newer version of the toolkit; so we should migrate people along. @pierceboggan thoughts? This would be my proposal:
> > Teams Toolkit will migrate your project to support the latest features in Teams Toolkit 2.7.2. This process will change the contents of the '.fx' folder. Backup this folder or put your project in source control if you want to revert your project back to Teams Toolkit 2.7.1.
> > (can you check that the version tags are correct?)
> 
> Thank you @timngmsft for your feedback. the message has updated, and the version start 2.7.0 will migration the project
> 
> **We will migrate your project to support the latest features in Teams Toolkit. This process will change the contents of the '.fx' folder. Backup this folder or put your project in source control if you want to revert your project back to Teams Toolkit version below 2.7.0**

The feature is still in "Insider" review.  Shall we call it out? Since actually turn off feature flag, there is no upgrade needed.

**Comment by wenytang-ms** [comment]
message updated

**Teams Toolkit will migrate your project to support the latest features in Teams Toolkit. This process will change the contents of the '.fx' folder. Backup this folder or put your project in source control if you want to revert your project back below Teams Toolkit 2.7.0.**

File: packages/fx-core/resource/strings.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
...
     "LocalDebugTenantConfirmNotice": "You already run local debug for the Teams app in another tenant '%s'. You can sign out %s and sign in with the matched M365 account to re-use the previous local debug resource. If you really want to switch tenant for local debug, you can delete the localSettings.json file and re-run local debug.",
     "MigrateSuccessNotice": "Your project is now initialized with Teams Toolkit! You can debug your application by pressing F5. If you wish to host your application in Azure, we recommend you to re-create your project directly using the latest Teams Toolkit.",
-    "MigrationToArmAndMultiEnvMessage": "We will update your project in order to continue using the latest Teams Toolkit. We recommend to initialize your workspace with git for better tracking file changes.",
+    "MigrationToArmAndMultiEnvMessage": "Teams Toolkit will migrate your project to support the latest features in Teams Toolkit. This process will change the contents of the '.fx' folder. Backup this folder or put your project in source control if you want to revert your project back below Teams Toolkit 2.7.0.",
```
Comment (timngmsft) [inline packages/fx-core/resource/strings.json:27]: Just one more change - "Teams Toolkit will migrate your project to support the latest features. This process ..."

**Review by timngmsft — CHANGES_REQUESTED** [review]

**Review by timngmsft — APPROVED** [review]

**Pull request merged**